### PR TITLE
Migrating and upgrading angular packages

### PR DESCRIPTION
**Pull request by BrendenMcGrew**
# To be Completed by the Pull Requester

### Checklist:

- [x] I have performed a self-review of my code to ensure that unintended code is not introduced.
- [x] I have commented my code, particularly in hard-to-understand areas to ensure that the next group of developers can understand my code.
- [x] I have added tests (if appropriate) that prove my fix/feature functions so that future modifications to the code do not break my additions.

### Description:

Upgraded any packages that wouldn't cause a breaking change

### Type of change (Pick Any)

Upgrading packages

### How Has This Been Tested?

N/A

---

# To be Completed By Reviewers

Reviewer # 01

- [x] I have performed a review of the code and checked for possible unintentional edits.
- [x] I concur with the amount of comments made.
- [x] I concur with the amount of tests made.

Reviewer # 02

- [x] I have performed a review of the code and checked for possible unintentional edits.
- [x] I concur with the amount of comments made.
- [x] I concur with the amount of tests made.

### CODE REVIEW

**Review by pmeskridge — APPROVED** [review]

File: apps/ods/src/app/responses/select-tags/select-tags.component.ts [hunk 1]
```diff
@@ -58,18 +58,25 @@ export class SelectTagsComponent {
...
       this.responsesStore.tagSuccess$.pipe(take(2)).subscribe((data) => {
         //Adds the tag to the chip display if it was successfully added to the database
         if (data) {
-          this.selectedTags?.push(input);
           // Clear the input values
           if (event.chipInput) {
             event.chipInput.clear();
           }
         }
+        //Checks that the data is false but not null.
```
Comment (yurisim) [inline apps/ods/src/app/responses/select-tags/select-tags.component.ts]: Do you need an else if? When is it null?

**Review by yurisim — CHANGES_REQUESTED** [review]

File: docs/how-tos.md [hunk 2]
```diff
@@ -2,6 +2,8 @@
 
 1. type `nx migrate latest` in the terminal to update the packages. This may take a few minutes. It will also make code changes so that the newly updated packages do not cause API conflicts. Follow the instructions in the terminal when the install completes. Verify that E2E tests pass then commit the changes.
 
-2. type `yarn upgrade-interactive --latest` use your spacebar and arrow keys to select only the green packages. After selecting all green packages press enter. Verify that E2E tests pass then commit the changes.
+2. If migrations are needed, make sure to run `yarn nx migrate --run-migrations`. ONLY IF migrations are needed.
```
Comment (yurisim) [inline docs/how-tos.md:5]: step 2 should be after "follow the instructions in the terminal"

**Review by yurisim — DISMISSED** [review]

File: apps/ods/src/app/responses/select-tags/select-tags.component.ts (hunk 1 above)
Comment (BrendenMcGrew) [reply apps/ods/src/app/responses/select-tags/select-tags.component.ts]: No, because I don't want it to do anything.

File: docs/how-tos.md (hunk 2 above)
Comment (BrendenMcGrew) [reply docs/how-tos.md:5]: Fixed by moving the steps after that to step 3.

**Review by REDRICK696 — APPROVED** [review]

**Pull request merged**